app/src/main/java/com/eventyay/organizer/core/main/FragmentNavigator.java

```diff
@@ -72,18 +68,6 @@ void loadFragment(int navItemId) {
             case R.id.nav_sell:
                 fragment = CreateOrderFragment.newInstance(eventId);
                 break;
-            case R.id.nav_attendees:
-                fragment = AttendeesFragment.newInstance(eventId);
-                break;
-            case R.id.nav_tickets:
-                fragment = TicketsFragment.newInstance(eventId);
-                break;
-            case R.id.nav_orders:
-                fragment = OrdersFragment.newInstance(eventId);
-                break;
-            case R.id.nav_event_settings:
-                fragment = EventSettingsFragment.newInstance(eventId);
-                break;
```

Comment:
Don't remove

Reply:
We have to keep them in the navigation drawer also?

Reply:
Updated.